src/Resources/app/administration/src/module/ui-feed-export/component/ui-feed-export-form/index.js

```diff
@@ -10,19 +10,42 @@ Component.register('ui-feed-export-form', {
             salesChannelValue: null,
             salesChannelLanguageValue: null,
             exportTypeValue: null,
-            typeSelectOptions: {
-                'products': 'Products',
-                'cms': 'CMS',
-                'brands': 'Brands'
-            }
+            typeSelectOptions: [],
         }
     },
 
     mixins: [
         Mixin.getByName('notification')
     ],
-
+    mounted () {
+        this.getExportTypeValues()
+    },
+    filters: {
+        capitalize: function (value) {
+            if (!value) return '';
+            value = value.toString();
+            return value.charAt(0).toUpperCase() + value.slice(1);
+        }
+    },
     methods: {
+        getExportTypeValues() {
+            const httpClient = Shopware.Service('syncService').httpClient;
+            let url = '_action/fact-finder/get-export-type-options';
```

Comment:
Could use `const` here because `url` is not supposed to be reassigned.
```suggestion
            const url = '_action/fact-finder/get-export-type-options';
```